BitwardenShared/Core/Platform/Services/StateService.swift

```diff
@@ -519,6 +525,11 @@ protocol StateService: AnyObject {
     ///
     func setIntroCarouselShown(_ shown: Bool) async
 
+    /// Sets the status of Learn generator Action Card.
+    ///
+    /// - Parameter status: The status of Learn generator Action Card.
+    ///
+    func setLearnGeneratorActionCardStatus(_ status: AccountSetupProgress) async
     /// Sets the status of Learn New Login Action Card.
```

Comment:
⛏️ Missing empty line:
```suggestion
    func setLearnGeneratorActionCardStatus(_ status: AccountSetupProgress) async

    /// Sets the status of Learn New Login Action Card.
```